openfecwebapp/config.py

```diff
@@ -28,7 +28,7 @@
 )
 
 features = {
-    'legal': bool(env.get_credential('FEC_FEATURE_LEGAL', '')),
+    'legal_statutes': bool(env.get_credential('FEC_FEATURE_LEGAL_STATUTES', '')),
```

Comment:
Would we want to just use the same flag name `features.legal`?  Minimizes updates to our readme docs.